es-ds-components/components/b-col.vue

```diff
@@ -1,3 +1,14 @@
+<script setup lang="ts">
```

Comment:
One thing I do like is these settings result in putting the `<script>` tag at the top of vue files